### DISCUSSION TITLE
A proxy server that doesn't suck

### DISCUSSION THREAD

**Discussion by xlak** [xlak/hapara-delete#3] [Q&A]
The Alphabetic exploit needs a proxy server to be useful. Public proxy servers are usually overloaded, so they won't work very well. I'm currently experimenting with using some free hosting providers to host proxies. However, most of these either turn out to have one or more of the following problems:

- High network latencies
- Needs to be accessed every 5 minutes to make it not go to sleep
- Randomly goes down and doesn't come back up until I go onto the dashboard
- Doesn't support WebSockets (breaks many apps such as Discord and online games)
- Blocked by certain filters.

Do you know of any reliable public proxy servers or free hosting providers?

**Comment by m1kecraft** [comment]
I use normally any that I find on the deploy section of proxies, but I would recommend railway.  The only downside is that the first few times you use it, it is confusing.  There are better ones, but they are blocked on school computers. (like codesandbox, replit, etc.)
